tools/build.ts

```diff
@@ -0,0 +1,60 @@
+import { spawn } from 'node:child_process'
+
+const order = [
```

Comment:
Was the optimal order determined manually, or was some sort of tooling used in generating this sequence?  
If second one - wold be cool add some comment about how to